polyfills/MutationObserver/tests.js

```diff
@@ -1,18 +1,22 @@
 /* Assertion bridge between web components test suite and expectjs */
 
+
 var expectRecord = function (src, model) {
 	Object.keys(model).forEach(function(key) {
-		expect(src[key]).to.eql(model[key]);
+		// proclaim(src[key]).to.eql(model[key]);
+    proclaim(src[key], model[key]);
```

Comment:
There are some indentation issues here, @hipkiss91 could you configure your editor to respect our .editorconfig settings?  cc @JakeChampion